pymc_marketing/mmm/tvp.py

```diff
@@ -103,52 +103,56 @@ def time_varying_prior(
     return pm.Deterministic(name, centered_f, dims=dims)
 
 
-def create_time_varying_intercept(
+def create_time_varying_gp_multiplier(
+    name: str,
```

Comment:
Might be better to just pass a subdictionary rather than having to add a name which creates a prefix. 

i.e. pass `model_config["intercept_tvp_config"]` instead of `model_config`
```python
# Example config
model_config = {
    "intercept_tvp_config": {
        "m": 200,
        "L": None,
        "eta_lam": 1,
        "ls_mu": None,
        "ls_sigma": 10,
        "cov_func": None,
    },
}
```

Reply:
Done!